docs/SFORMS_WORKFLOW_DOC.md

```diff
@@ -0,0 +1,22 @@
+# S-Forms publishing workflow documentation
+
+S-Forms follows the [semantic versioning](https://docs.npmjs.com/about-semantic-versioning) recommendations for new **releases**, and a SHA based ID for new **pre-releases**. The workflow is described below:
+
+
+## Publish new release
+
+New releases should be done **only** on the **master** branch. Use the `release:<patch | minor | major>` to publish a new release. The script will automatically bump the version, create a new git tag and publish the library to npm. Refer to [semantic versioning](https://docs.npmjs.com/about-semantic-versioning) to determine which script to use.
```

Comment:
```suggestion
New releases are done **only** on the **master** branch. Use the `npm run release:<patch | minor | major>` to publish a new release. The script will automatically bump the version, create a new git tag and publish the library to npm. Refer to [semantic versioning](https://docs.npmjs.com/about-semantic-versioning) to determine which script to use.
```

Reply:
Thus if I understand it correctly, you must also commit first ... I would put this fact into the comment (something like make sure that everything is committed)

If it does git tag it also pushes it right ? I would add it there as well if so ...